internal/pkg/aws/cloudformation/testdata/parse/env.yaml

```diff
@@ -0,0 +1,740 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+Metadata:
+  Version: 'v1.1.0'
+
+Parameters:
+  AppName:
+    Type: String
+
+  EnvironmentName:
+    Type: String
+
+  ALBWorkloads:
+    Type: String
+    Default: ""
+
+  ToolsAccountPrincipalARN:
+    Type: String
+
+  AppDNSName:
+    Type: String
+    Default: ""
+
+  AppDNSDelegationRole:
+    Type: String
+    Default: ""
+
+Conditions:
+  CreateALB:
+    !Not [!Equals [ !Ref ALBWorkloads, "" ]]
+  DelegateDNS:
+    !Not [!Equals [ !Ref AppDNSName, "" ]]
+  ExportHTTPSListener: !And
+    - !Condition DelegateDNS
+    - !Condition CreateALB
+
+Resources:
+  VPC:
+  # Virtual private cloud on 2 availability zones to control network boundaries.
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock: 10.0.0.0/16
+      EnableDnsHostnames: true
+      EnableDnsSupport: true
+      InstanceTenancy: default
+      Tags:
+        - Key: Name
+          Value: !Sub 'copilot-${AppName}-${EnvironmentName}'
+
+  PublicRouteTable:
+  # Routing table for services to talk with each other.
+    Type: AWS::EC2::RouteTable
+    Properties:
+      VpcId: !Ref VPC
+      Tags:
+        - Key: Name
+          Value: !Sub 'copilot-${AppName}-${EnvironmentName}'
+
+  DefaultPublicRoute:
+    Type: AWS::EC2::Route
+    DependsOn: InternetGatewayAttachment
+    Properties:
+      RouteTableId: !Ref PublicRouteTable
+      DestinationCidrBlock: 0.0.0.0/0
+      GatewayId: !Ref InternetGateway
+
+  InternetGateway:
+  # Internet gateway to connect the network to the internet.
+    Type: AWS::EC2::InternetGateway
+    Properties:
+      Tags:
+        - Key: Name
+          Value: !Sub 'copilot-${AppName}-${EnvironmentName}'
+
+  InternetGatewayAttachment:
+    Type: AWS::EC2::VPCGatewayAttachment
+    Properties:
+      InternetGatewayId: !Ref InternetGateway
+      VpcId: !Ref VPC
+
+  PublicSubnet1:
+  # A public subnet in your first AZ for internet facing services.
```

Comment:
Do we need to write out "availability zone (AZ)"? Or is that abbreviation commonly known?

Reply:
This file is only for testing (anything under ["testdata"](https://dave.cheney.net/2016/05/10/test-fixtures-in-go)) so I've put whatever I could think of at the time.

I'll send a separate PR adding officially these comments to our templates after this